testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.export.tests.js

```diff
@@ -4,10 +4,19 @@ import "ui/data_grid/ui.data_grid";
 
 import $ from "jquery";
 import { excel as excelCreator } from "client_exporter";
+import excel_creator from "client_exporter/excel_creator";
+import JSZipMock from "../../helpers/jszipMock.js";
+
+const oldJSZip = excel_creator.ExcelCreator.JSZip;
 
 QUnit.testStart(function() {
     var markup = '<div id="dataGrid"></div>';
     $("#qunit-fixture").html(markup);
+    excel_creator.ExcelCreator.JSZip = JSZipMock;
```

Comment:
Use this code inside the "beforeEach" and "afterEach" methods of module settings. For example, please see the \testing\tests\DevExpress.ui.widgets\accordion.tests.js